### PR TITLE
[#106][GitHub] Skip CI for docs changes only

### LINKED ISSUE #106: [CI] Skip CI for docs changes only
**What feature are your requesting?**

- This feature is to avoid CI builds for documentation changes 

**What benefits do you foresee from the feature you are requesting?**

- Avoid : time & resource utilization in building code for changes that touch only documentation.

**Potential solution/ideas?**

https://github.com/google/EarlGrey/pull/383/files/3b38a5dea36a88aba42a42931e77a7c5429a1837

**Additional context**

### PR DESCRIPTION
**Pull request by ameya-pandilwar**
[CI] Skip CI for docs changes only #106

Make sure you have checked all steps below.

### GitHub Issue
Fixes #106 

### Checklist:

- [ ] This pull request updates the documentation
- [ ] This pull request changes library dependencies
- [x] Title of the PR is of format (example) : [#25][Github] Add Pull Request Template

### What is the purpose of this pull request?

Skip CI for docs changes only

### Commit Guidelines
- [x] My commits all reference GH issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by ameya-pandilwar** [comment]
@Dee-Pac / @anishanainani -- can one of you please review and merge this?

**Comment by Dee-Pac** [comment]
@ameya-pandilwar  Thanks for raising this PR.

Could you please share a snippet of how this PR was tested ? Any snapshots will be helpful.

**Comment by ameya-pandilwar** [comment]
> @ameya-pandilwar Thanks for raising this PR.
> 
> Could you please share a snippet of how this change has been validated ? Any snapshots will be helpful.

@Dee-Pac -- I triggered a sample run on a PR with only documentation changes --
Screen Shot 2020-06-30 at 10 33 39 PM

For Reference : https://travis-ci.com/github/ameya-pandilwar/gimel/jobs/356172832

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,14 @@ dist: trusty
 # Language - we have scala, java & some python files
 language: generic
 
+before_install:
+  - |
+    if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(.md)|(.png)|(.pdf)|(.html)|^(LICENSE)|^(docs)'
+    then
+      echo "Only doc files were updated, not running the CI."
+      exit
```
Comment (Dee-Pac) [inline .travis.yml:27]: @ameya-pandilwar  - Could you please explicitly make this `exit 0`?

Comment (ameya-pandilwar) [reply .travis.yml]: @Dee-Pac -- done!

**Pull request merged**